network/p2p/libp2pNode_test.go

```diff
@@ -2,69 +2,23 @@ package p2p
 
 import (
 	"context"
-	"fmt"
-	"net"
-	"os"
 	"testing"
 	"time"
 
-	golog "github.com/ipfs/go-log"
-	addrutil "github.com/libp2p/go-addr-util"
 	"github.com/libp2p/go-libp2p-core/network"
 	"github.com/libp2p/go-libp2p-core/peer"
 	"github.com/libp2p/go-libp2p-core/peerstore"
-	"github.com/multiformats/go-multiaddr"
-	manet "github.com/multiformats/go-multiaddr/net"
-	"github.com/rs/zerolog"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"github.com/stretchr/testify/suite"
 
-	fcrypto "github.com/onflow/flow-go/crypto"
 	"github.com/onflow/flow-go/model/flow"
-	"github.com/onflow/flow-go/module/metrics"
-	"github.com/onflow/flow-go/network/mocknetwork"
-	"github.com/onflow/flow-go/network/p2p/dns"
 	"github.com/onflow/flow-go/utils/unittest"
 )
 
-// Workaround for https://github.com/stretchr/testify/pull/808
-const tickForAssertEventually = 100 * time.Millisecond
-
-// Creating a node fixture with defaultAddress lets libp2p runs it on an
-// allocated port by OS. So after fixture created, its address would be
-// "0.0.0.0:<selected-port-by-os>
-const defaultAddress = "0.0.0.0:0"
-
-var sporkID = unittest.IdentifierFixture()
-
```

Comment:
Most of these are common between several tests in this `p2p` package, and hence moved to `p2p/fixtures.go`